cost/cloud_cost_anomaly_alerts/cloud_cost_anomaly_alerts.pt

```diff
@@ -161,13 +181,39 @@ datasource "ds_get_anomalies" do
 end
 
 datasource "ds_filter_anomalies" do
-  run_script $js_filter_anomalies, $ds_get_anomalies, $param_min_spend, rs_org_id, $param_days, $param_dimensions, $param_cost_metric, $param_anomaly_limit, f1_app_host
+  run_script $js_filter_anomalies, $ds_get_anomalies, $param_min_spend, rs_org_id, $param_days, $param_dimensions,$ds_filter_dimensions, $param_cost_metric, $param_anomaly_limit, f1_app_host
 end
 
 ###############################################################################
 # Scripts
 ###############################################################################
 
+script "js_filter_dimensions", type: "javascript" do
+  parameters "param_dimensions", "ds_dimensions"
+  result "filtered_dimensions"
+  code <<-EOS
+    var filtered_dimensions = {
+      dimensions_ids: [],
+      dimensions_names: [],
```

Comment:
optional: we are in context of dimensions (filtered_dimensions), Can we rename it to id, names, invalid_ids instead of dimension_ids, dimension_names, invalid_dimensions ?

Reply:
Ok